packages/contracts-bedrock/scripts/deploy/Deploy.s.sol

```diff
@@ -407,21 +407,47 @@ contract Deploy is Deployer {
         deployMulticall3();
     }
 
+    function preInitializeOptimismPortalBalance() public broadcast {
+        address optimismPortalProxy = mustGetAddress("OptimismPortalProxy");
+        address storageSetter = mustGetAddress("StorageSetter");
+
+        // NOTE: the storage slot index should stay the same across versions
+        // (OptimismPortal, OptimismPortal2, ...)  since slot spacers are used
+        // for legacy storage variables.
+        // We also assert correctness in a downstream ChainAssertion,
+        // so changing slot numbers should get detected for coming versions.
+        uint256 balanceStorageSlot = 61; // slot of _balance variable
```

Comment:
Is this slot same for both OptimismPortal1 & OptimismPortal2 ?

Reply:
Yes, it seems they introduce some empty dummy slots in the new contract exactly for slot backwards compatibility.
But please go ahead and double check this, as getting the number right is important ;)